manifests/s/SanfordLP/DYMOConnect/1.4.3/SanfordLP.DYMOConnect.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: SanfordLP.DYMOConnect
+PackageVersion: 1.4.3

```

```suggestion
PackageVersion: 1.4.3.131
```
You will need to update the name of the version folder as well when changing this